src/button/button.module.scss

```diff
@@ -0,0 +1,52 @@
+.buttonColor {
+    background-color: #F7F2F2;
+    color: #121616;
```

Comment:
Maybe you can centralize all colors in one file and import these colors in all module.scss files.

Reply:
I agree with cacho! You will see that is common in every project to have a file named "ColorPalette" for example, to define all the colors that will be used in the app and then import the color you need in each screen from that file.